src/utils/graph/index.js

```diff
@@ -0,0 +1,104 @@
+import { offsetNode, offsetEdge } from './common';
+import { layout } from './layout';
+import { routing } from './routing';
+
+const defaultOptions = {
```

Comment:
I wonder whether this object should go in /src/config.js? 🤔

Reply:
These are the default values for the function / module, rather than a user config, so I'd say it's best they stay with the module if that makes sense.